internal/config/docs.toml

```diff
@@ -78,34 +72,14 @@ FallbackPollInterval = '30s' # Default
 # **ADVANCED**
 # Ideally, you should use a container orchestration system like [Kubernetes](https://kubernetes.io/) to ensure that only one Chainlink node instance can ever use a specific Postgres database. However, some node operators do not have the technical capacity to do this. Common use cases run multiple Chainlink node instances in failover mode as recommended by our official documentation. The first instance takes a lock on the database and subsequent instances will wait trying to take this lock in case the first instance fails.
 #
-# By default, Chainlink nodes use the `dual` setting to provide both advisory locks and lease locks for backward and forward compatibility. Using advisory locks alone presents the following problems:
-#
 # - If your nodes or applications hold locks open for several hours or days, Postgres is unable to complete internal cleanup tasks. The Postgres maintainers explicitly discourage holding locks open for long periods of time.
-# - Advisory locks can silently disappear when you upgrade Postgres, so a new Chainlink node instance can take over even while the old node is still running.
-# - Advisory locks do not work well with pooling tools such as [pgbouncer](https://www.pgbouncer.org/).
-# - If the Chainlink node crashes, an advisory lock can hang around for up to several hours, which might require you to manually remove it so another instance of the Chainlink node will allow itself to boot.
 #
-# Because of the complications with advisory locks, Chainlink nodes with v1.1.0 and later support a new `lease` locking mode. This mode might become the default in future. The `lease` locking mode works using the following process:
+# Because of the complications with advisory locks, Chainlink nodes with v2.0 and later only support `lease` locking mode. The `lease` locking mode works using the following process:
```

Comment:
I wouldn't remove advisory locking completely, I think we just want to make `lease` the new default.

Reply:
Did you change your mind, or did I misunderstand? https://app.shortcut.com/chainlinklabs/story/45329/deprecate-database-locking-mode-advisory